app/table-config/census/household-type.js

```diff
@@ -5,11 +5,11 @@ export default [
   { data: 'nfmhhu65', title: 'Householder under 65 years', indent: 1 },
   { data: 'nfmlvgaln', title: 'Householder living alone', indent: 1 },
   { data: 'nflva65pl', title: 'Householder 65 years and over', indent: 2 },
-  { divider: true },
+  { divider: true,},
   { data: 'hh_2', title: 'Total households', highlight: true },
   { data: 'mrdcplhh', title: 'Married couple household' },
   { data: 'mrdch18', title: 'With own children under 18', indent: 1 },
-  { data: 'cocplhh', title: 'Cohabiting couple household' },
+  { data: 'cocplhh', title: 'Cohabitng couple household' },
```

Comment:
Small typo here

Reply:
yeah, I am going to make a separate PR with the correction. For now, I need to revert the entire PR